recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - cymem 2.0.6
-    - cython 0.29.32
+    - cython 0.29
     - cython-blis 0.7.9
     - murmurhash 1.0.7
     - numpy {{ numpy }}
```

Comment:
it looks like the run requirements don't match with the host ones, e.g. you can run with lower versions than the build versions for the four cython related libaries. could use `pin_compatible`